components/webui/imports/ui/SearchView/SearchView.jsx

```diff
@@ -155,11 +157,15 @@ const SearchView = () => {
 
         Meteor.subscribe(Meteor.settings.public.AggregationResultsCollectionName, {
             aggregationJobId: aggregationJobId,
+            isActivelyPolling: resultsMetadata.lastSignal === SEARCH_SIGNAL.REQ_QUERYING,
```

Comment:
```suggestion
            isActivelyPolling: SEARCH_SIGNAL.REQ_QUERYING === resultsMetadata.lastSignal,
```

The linter should have fixed this.